styleguide/modules/ROOT/pages/placeholders.adoc

```diff
@@ -1,9 +1,32 @@
 = Code Placeholders
 
-When you need to add a placeholder value to a xref:examples.adoc[code example]: 
+The type of placeholder you add to code depends on the type of xref:examples.adoc[code example] you're including. 
+
+== curl and Shell Placeholders
```

Comment:
cURL

Reply:
I was surprised to find out recently that they've renamed cURL to curl.
Had to change it in a few places in the docs.